dotnetapp-buildcontainer/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Capture the current build and app image, if any, so that we can clean them up when we are done.
+currentBuildImage=$(docker images build-image -q)
+currentAppImage=$(docker images dockerapp -q)
+
+docker build -t build-image -f Dockerfile.build .
+
+# Create a container from our build image so that we can copy the contents out.
+docker create --name build-container build-image
+# Copy the contents of the /out directory out so that we can build our app image.
+docker cp build-container:/out ./out
+# Build the application image.
+docker build -t dockerapp -f ./out/Dockerfile ./out
```

Comment:
I misunderstood what you were asking about. For some reason I thought you were asking about copying the Dockerfile into the publish output folder, not adding it to the image. Yes I can change that.